modal/index.js

```diff
@@ -122,6 +122,19 @@ Modal.prototype.setVisible = function(enabled) {
   }
 
   var lightboxEl = document.querySelector('.' + this.config.className);
+
+  var enterClass = this.config.className + '--enter';
+  var exitClass = this.config.className + '--exit';
+  window.setTimeout(function() {
+    if(enabled) {
```

Comment:
Small nit - should have a space in front of the `(` -- also should there be a timeout duration or is null OK? Thanks for this enhancement - feel free to merge in once addressing!